### PR TITLE
feat(useAverage): new function

### PR DESCRIPTION
**Pull request by colgin**
### Additional context
#1812 

---

### What is the purpose of this pull request? 

- [ ] Bug fix
- [x] New Feature
- [ ] Documentation update
- [ ] Other

### Before submitting the PR, please make sure you do the following

- [x] Read the [Contributing Guidelines](https://github.com/vueuse/vueuse/blob/main/CONTRIBUTING.md).
- [x] Read the [Pull Request Guidelines](https://github.com/vueuse/vueuse/blob/main/packages/guidelines.md).
- [x] Check that there isn't already a PR that solves the problem the same way to avoid creating a duplicate.
- [ ] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).
- [x] Ideally, include relevant tests that fail without this PR but pass with it.

### CODE REVIEW

**Comment by webfansplz** [comment]
Please send PR against `next` branch

File: packages/math/useAverage/index.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+category: '@Math'
+---
+
+# useAverage
+
+Get the average value of a branch of numbers
```
Comment (antfu) [inline packages/math/useAverage/index.md:7]:
```suggestion
Get the average value of an array of numbers
```

**Review by antfu — APPROVED** [review]
LGTM. I like the way we support nested refs.

**Comment by colgin** [comment]
oh, wait a minute

**Comment by colgin** [comment]
> LGTM. I like the way we support nested refs.

it seems that you help me to fixed the doc, so I don't need do anyting on this pr, right?

**Pull request merged**